app/ui-message/client/message.css

```diff
@@ -0,0 +1,3 @@
+pre {
+    font-family: -apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,Oxygen,Ubuntu,Cantarell,'Helvetica Neue','Apple Color Emoji','Segoe UI Emoji','Segoe UI Symbol','Meiryo UI',Arial,sans-serif;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4928441)